components/nimbus/src/stateful/persistence.rs

```diff
@@ -24,9 +24,18 @@ use std::path::Path;
 //
 // ⚠️ Warning : Altering the type of `DB_VERSION` would itself require a DB migration. ⚠️
 pub(crate) const DB_KEY_DB_VERSION: &str = "db_version";
-pub(crate) const DB_VERSION: u16 = 2;
+pub(crate) const DB_VERSION: u16 = 3;
 const RKV_MAX_DBS: u32 = 6;
 
+pub(crate) const DB_KEY_EXPERIMENTS_USER_PARTICIPATION: &str = "user-opt-in-experiments";
+pub(crate) const DB_KEY_ROLLOUTS_USER_PARTICIPATION: &str = "user-opt-in-rollouts";
```

Comment:
experiment should be singular "experiment participation" vs "experiments". likewise with rollout